cart.js

```diff
@@ -1,20 +1,11 @@
-const navCart = document.getElementById("nav__cart");
 const cartMenu = document.getElementById("nav__cart-menu");
 
-document.addEventListener("click", function(event){ 
-	  if (event.target==navCart) {
-      if(!hamburgerMenu.classList.contains("--hidden")) {
-        hamburgerMenu.classList.toggle("--hidden");
-       hamburgerIcon.classList.toggle("change");
-      }
-     cartMenu.classList.toggle("--hidden");
-    }
-    else {
-      if(!cartMenu.contains(event.target)){
-        if (!cartMenu.classList.contains("--hidden")) {
-         cartMenu.classList.toggle("--hidden");
-        }
-      }
-    }
+export function cartControl() {
+  cartMenu.classList.toggle("--hidden");
+}
+
+export function cartHide() {
```

Comment:
Does this function do just hide? 
I see what in function body happen some check actions and calls another function. Advise calling it as `toogleCartControl` or finding another better name that tells what do this function.